extensions/amp-story/0.1/amp-story-auto-ads.js

```diff
@@ -226,8 +246,58 @@ export class AmpStoryAutoAds extends AMP.BaseElement {
     const attributes = /** @type {!JsonObject} */ (Object.assign({},
         defaultAttrs, configAttrs));
 
-    return createElementWithAttributes(
+    const adElement = createElementWithAttributes(
         document, 'amp-ad', attributes);
+
+    this.currentAdElement_ = adElement;
+    return adElement;
+  }
+
+
+  /**
+   * Validate ad-server response has requirements to build outlink
+   * @param {!Element} adPageElement
+   */
+  maybeCreateCtaLayer_(adPageElement) {
+    // if making a CTA layer we need a button name & outlink url
+    const ctaUrl = this.currentAdElement_.getAttribute(CTA_URL_ATTR);
+    const ctaType = this.currentAdElement_.getAttribute(CTA_TYPE_ATTR);
+
+    if (!ctaUrl || !ctaType) {
+      dev().error(TAG, 'Both "vars.ctaType" & "vars.ctaUrl" ' +
```

Comment:
this should not be an error as branding ads do not need CTA button

Reply:
spoke offline. Changed error type